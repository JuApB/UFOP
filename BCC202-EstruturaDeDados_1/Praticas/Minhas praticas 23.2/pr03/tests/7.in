408
5270 79
3039 75
988 63
15193 81
4845 77
9099 30
17946 6
5848 37
14894 94
948 30
8906 38
16462 37
13426 79
18035 53
1908 69
16920 15
5951 79
2543 74
18881 80
6999 50
10347 29
16001 28
16178 41
6073 48
10298 45
17773 73
11417 55
11574 6
8222 19
15144 97
19386 41
14758 42
15141 47
9443 22
4112 33
8622 39
9156 13
19371 43
18244 78
19653 19
2671 63
18614 53
6728 74
9952 21
4756 90
9672 26
3999 21
14531 9
916 60
2895 32
6466 23
10062 53
18080 34
1243 35
17190 46
6041 98
18573 3
12036 59
3322 23
9145 80
9749 45
3028 30
11545 69
3588 27
3374 59
11656 63
4191 94
3866 28
11502 90
17310 72
10944 21
2546 9
18843 38
15466 44
8984 90
9673 1
18079 74
12525 51
11177 21
3107 86
9904 48
4394 97
5907 88
16668 57
4588 44
8088 10
15672 29
444 9
11622 83
8674 33
18034 16
16826 52
1205 14
1842 49
8495 10
5750 83
4677 13
10904 94
1751 15
18272 3
14028 82
15982 94
10871 71
17542 75
16871 48
3643 34
17702 66
11776 7
3882 92
5312 33
12921 70
4960 58
10706 37
8892 74
7831 75
428 5
9094 31
4415 57
19462 0
3961 86
3580 72
4396 58
1651 51
19128 55
11863 57
395 75
3610 6
9484 26
6031 84
8418 31
12610 41
16225 96
14870 72
13481 66
17828 63
10884 83
1538 40
11162 28
6843 21
7683 40
8478 22
12041 34
16997 84
5829 52
16422 53
4882 21
9586 86
6729 13
12095 92
18094 43
19277 7
3871 4
18960 32
16733 24
5152 51
6295 21
10964 54
16728 70
14121 28
13174 98
9372 69
11348 40
19465 80
5137 93
16626 99
10649 54
5058 44
3822 90
15883 67
8785 62
722 16
12657 41
1123 49
18705 99
14529 67
12535 93
14270 97
12096 95
2325 67
1244 21
10056 87
11192 30
10978 13
5862 99
8315 42
19447 64
6221 92
7563 26
12293 43
12580 78
4953 90
2715 22
5804 38
9451 49
6435 24
13972 40
5037 28
16745 48
12424 48
13237 69
12012 13
2526 10
11746 96
338 8
8685 24
12543 9
18740 75
9832 12
17826 15
100 5
4143 37
2783 96
17253 41
1482 23
6437 20
17937 37
3558 39
10044 3
16090 46
19742 18
14221 72
13464 19
15295 0
17521 65
4767 68
5105 59
11475 87
19317 55
9689 0
4557 14
8147 95
16014 79
10540 78
10709 30
3825 79
17906 91
19182 53
16136 46
11217 26
13465 45
8518 69
857 60
14939 30
7312 8
14759 81
799 68
19708 54
3454 55
16945 22
17687 21
8857 90
14328 52
13725 27
10337 53
14802 59
10167 82
14058 95
19213 95
10374 30
10319 24
2124 45
11661 22
11912 70
7433 30
4525 74
9356 76
17603 81
2964 62
11920 42
6364 24
4794 96
15622 95
18445 10
10334 12
1132 8
19169 84
6166 48
13371 39
2864 21
3437 96
124 91
16976 94
19138 92
6647 3
5506 19
9565 26
15803 17
13615 76
3472 15
2382 99
15028 84
9079 68
4243 42
11673 55
17371 69
5957 24
8095 76
10721 54
13418 91
6829 45
5896 28
8914 82
9555 72
372 88
5309 19
9208 97
14622 85
15108 37
19800 68
2172 80
16561 83
12256 16
14623 10
6597 23
5949 56
15652 50
18723 43
15543 27
15552 53
1066 54
8670 83
3957 98
11547 19
4304 25
11037 63
15443 4
12673 37
17351 5
9381 8
16540 22
1800 73
2484 68
2336 22
13680 78
9873 63
15139 11
14244 17
14245 47
9440 6
2804 0
14661 2
7880 23
7441 67
7756 80
9973 73
12563 58
3234 93
586 87
9776 43
1671 23
10502 28
7524 45
9758 4
1728 10
14922 0
14402 19
2485 12
12222 20
9591 30
12905 86
15723 56
4945 79
18536 77
12894 4
540 72
16987 16
17722 39
16152 11
4144 38
12927 69
14440 70
1337 5
3497 66
8028 22
12337 3
1582 50
2223 27
2723 52
602 28
13309 91
8464 71
7024 36
10569 82
9038 81
9300 42
15617 73
11033 4
8174 75
5696 80
7284 77
18490 71
19948 3
3964 30
6549 17
3094 82
1570 56
6679 83
4356 19
16375 92
19273 16
1025 22
15101 27
11960 71
4027 74
11210 22
19936 86
19761 59
7984 51
0