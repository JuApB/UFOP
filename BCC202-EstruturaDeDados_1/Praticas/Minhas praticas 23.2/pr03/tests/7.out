Turma 1
16626 18705 5862 2382 
